etcdserver/api/membership/cluster.go

```diff
@@ -825,6 +834,22 @@ func detectDowngrade(cv *semver.Version, nextClusterVersionCompatible bool) erro
 	return nil
 }
 
+// IsValidVersionChange checks the two scenario when version is valid to change:
+// 1. Downgrade: cluster version is 1 minor version higher than local version,
+// cluster version should change.
+// 2. Cluster start: when not all members version are available, cluster version
+// is set to MinVersion(3.0), when all members are at higher version, cluster version
+// is lower than local version, cluster version should change
+func IsValidVersionChange(cv *semver.Version, lv *semver.Version, nextClusterVersionCompatible bool) bool {
```

Comment:
Note both versions are cluster versions. The second parameter name `lv` is misleading, although it's the same value as server local binary version in single node cluster.
```suggestion
func IsValidClusterVersionChange(verFrom *semver.Version, verTo *semver.Version, nextClusterVersionCompatible bool) bool {
```

Reply:
Right, suggestion is good. Please note that this code is take from main branch :P 

https://github.com/etcd-io/etcd/blob/e246bb8d4dfa61bd28e86a3683970379142b87f2/server/etcdserver/version/downgrade.go#L68-L76

Reply:
So is it more important to keep the code consistency or to address the new comments?

Reply:
Fix the main firstly, then fix this PR.

Reply:
Or please add a followup. But I prefer to fix main firstly, then fix this PR. Because lots of followups were forgotten.

Reply:
done